### PR TITLE
pragmarc 1.0.0

### PR DESCRIPTION
**Pull request by bracke**
Created via `alr publish` with `alr 2.0.1`

### CODE REVIEW

**Comment by mosteo** [comment]
Hi @bracke , have you agreed with @jrcarter on the versioning numbers here? I think he's simply using the date for releases.

**Comment by bracke** [comment]
No, we have not talked about versioning.
Would a date even be useable in Alire?

**Comment by zertovitch** [comment]
It is possible to use a date. For instance here:
https://github.com/alire-project/alire-index/blob/stable-1.3.0/index/ma/mathpaqs/mathpaqs-20230121.0.0.toml

**Comment by mosteo** [comment]
> No, we have not talked about versioning. Would a date even be useable in Alire?

Then, I'd very much prefer if the version matched, as @zertovitch exemplified, the tag from which this release is taken (assuming you're using one of @jrcarter's tagged releases).

**Comment by bracke** [comment]
>Then, I'd very much prefer if the version matched, as @zertovitch exemplified,
> the tag from which this release is taken (assuming you're using one of @jrcarter's tagged releases).

I'm using the default branch, which seems to be the tag v20240323.
I have changed the version to v20240323.

File: index/pr/pragmarc/pragmarc-1.0.0.toml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+name = "pragmarc"
+description = "The PragmAda Reusable Components"
+version = "v20240323"
```
Comment (mosteo) [inline index/pr/pragmarc/pragmarc-1.0.0.toml:3]: The `v` should be removed, and the file renamed to match the version, to `pragmarc-20240323.0.0.toml`.

**Review by mosteo — CHANGES_REQUESTED** [review]

**Pull request closed without merging**